switchboard/manager.py

```diff
@@ -33,7 +37,7 @@ def nested_config(config):
     return cfg
 
 
-def configure(config={}, nested=False):
+def configure(config={}, nested=False, request=None, user=None):
```

Comment:
Yeah, honestly, this (passing user and request into configure) is a hack right now, until I can code up a more robust API for defining a context (i.e. the data that a switch has to work with to determine if it should be active). Internally, user (assigned to `context['user']`) is only used when versioning a switch. Externally, it can be used (as we do) by custom conditions to turn switches on/off for certain users, groups, permissions, etc.